lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -141,18 +141,15 @@
 
     .action-label {
       @extend %t-copy-sub2;
-      display: inline-block;
+      display: none;
```

Comment:
@clrux Yep, this appears only on hover and focus. The previous implementation hid the button by making its opacity 0, so I changed it to just use display: none.